tests/atest/rules/errors/parsing_error/test_rule.py

```diff
@@ -3,7 +3,7 @@
 
 class TestRule(RuleAcceptance):
     def test_rule(self):
-        self.check_rule(expected_file="expected_output.txt", target_version=">=7")
+        self.check_rule(expected_file="expected_output_rf7.txt", target_version=">=7")
```

Comment:
Example of test that didn't run for 7.0a1 -> 7.0rc1 and this typo wasn't discovered